yarn.lock

```diff
@@ -23604,6 +23604,11 @@ react-native-confirm-device-credentials@^1.0.0:
   version "4.0.2"
   resolved "git://github.com/celo-org/react-native-contacts#4989b0b4df8bf84acf1979643af243b51453a504"
 
+"react-native-contacts@https://github.com/celo-org/react-native-contacts#4989b0b":
```

Comment:
Why did this change if we already have this dep in the mobile package?
Maybe it's because the link format isn't the same in react-components vs mobile. Please align the two

Reply:
hmn i thought i copied it over

Reply:
iv updated it to match exactly what is in mobile app